index.bs

```diff
@@ -312,23 +308,30 @@ A [=/font table=] has a <dfn>tag</dfn>, which is a {{ByteString}} of length 4, d
 
 A [=/font representation=] has a <dfn for="font representation">name table</dfn>, which is the [=/font table=] in its [=font representation/table list=] with [=font table/tag=] \``name`\`.
 
-The [=font representation/name table=] has a [=/map=] <dfn for="name table">names</dfn>, which is a mapping from an {{unsigned short}} to a [=/localized string table=].
+The [=font representation/name table=] has a [=/map=] <dfn for="name table">names</dfn>, which is a mapping from an {{unsigned short}} (a <dfn>name ID</dfn>) to a [=/localized string=]. [=/Name IDs=] are defined as part of the [[!OPENTYPE]] <a href="https://docs.microsoft.com/en-us/typography/opentype/spec/name">naming table</a> specification.
+
+A <dfn>localized string</dfn> is a [=/map=] from a BCP 47 language tag to a {{USVString}}. [[BCP47]]
```

Comment:
I'd prefer a term that implies that it's a more complicated structure than just a string, like the original "localized string table".

Reply:
I was trying to avoid "table" (also overloaded, e.g. this is all within the "names table"). Bikeshedding on term here appreciated. How is "localized string map" ?